README.md

```diff
@@ -1,3 +1,27 @@
+# OMI Python Script Provider
+
+## How to build
+
+### Dependencies to build OMI Python Script Provider:
+- Install gcc and g++
+  - Redhat/CentOS: `sudo yum install gcc gcc-c++`
+  - Debian/Ubuntu OS: `sudo apt-get install gcc g++`

```

Comment:
Does it work with any gcc version or only with some specific versions? Might be useful to include also the versions which are tested in my opinion.